src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -128,15 +127,14 @@ internal void OnBeforeWrite(TokenCacheNotificationArgs args)
                     //this should not happen but we have this as a safe guard
                     //against multiple matches.
                     IList<AccessTokenCacheItem> accessTokenItemList = new List<AccessTokenCacheItem>();
-                    AccessTokenCacheItem atItem = null;
                     foreach (var accessTokenString in TokenCacheAccessor.GetAllAccessTokensAsString())
                     {
-                        atItem = JsonHelper.DeserializeFromJson<AccessTokenCacheItem>(accessTokenString);
-                        if (atItem.ClientId.Equals(ClientId) &&
-                            atItem.Authority.Equals(requestParams.Authority.CanonicalAuthority) &&
-                            atItem.Scope.ScopeIntersects(accessTokenCacheItem.Scope))
+                        AccessTokenCacheItem token = JsonHelper.DeserializeFromJson<AccessTokenCacheItem>(accessTokenString);
```

Comment:
The variable name `atItem` gets flagged by StyleCop as hungarian notation.  I know it's not technically the case, but I figured it would be easy enough to rename it to something more descriptive.  Is there a different unabbreviated variable name you would prefer?